### PR TITLE
New: Concept Exercise Pacman Rules

### PR DESCRIPTION
**Pull request by pwadsworth**
## Work in Progress

Adds skeleton files for implementing concept exercise Pacman-Rules.

### CODE REVIEW

**Comment by pwadsworth** [comment]
Oops...

File: exercises/concept/annalyns-infiltration/.docs/hints.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Hints
+
+## General
+
+- There are three boolean operators to work with boolean values: `not`, `&&`, and `||`.
+- Multiple operators can be combined in a single expression.
+
+## 1. Check if a fast attack can be made
+
+- The boolean operators can also be applied to boolean parameters.
```
Comment (MatthijsBlom) [inline exercises/concept/annalyns-infiltration/.docs/hints.md:10]: What do you mean by this? I am confused by both «parameters» and «also».

Comment (pwadsworth) [reply exercises/concept/annalyns-infiltration/.docs/hints.md:10]: Generally, operators also apply to arguments. In this case though the text is just left over from the Annalyn's exercise. Change or delete as needed to fit the narrative of Pacman Rules.

**Comment by pwadsworth** [comment]
@MatthijsBlom I am moving all of this to draft PR #1093. You should be able to push there.

**Pull request closed without merging**